examples/hello_triangle_c/CMakeLists.txt

```diff
@@ -3,13 +3,20 @@ cmake_minimum_required(VERSION 3.11b)
 project(hello_triangle)
 
 set(TARGET_NAME hello_triangle)
-
+set(feature-native)
```

Comment:
where do we set the value to "feature-native"?

Reply:
Well that can actually be done by doing `cmake .. -Dfeature-native=dx12`, that was the only way I have found to do it now. If there is a better alternative, please tell me so and I will correct it.

Reply:
Maybe we could name it `BACKEND` and require it (if no value was passed then fail). The purpose would be to match whichever `BACKEND` that libwgpu-native was built with originally (ideally this would be a list of backends, but we could do that in a future PR).

Reply:
I agree, but for now it will be only used for dx12, so the require would not be necessary. I do not know if we have any information about what backend lib was built, so we would have to investigate that in what I propose a future issue.

Reply:
I don’t think we need to try to derive it - in Travis CI we could just pass it directly, and we can document it for people running cmake manually.

I agree it’s not a big deal right now, but it’s better to be consistent and require it than special-case dx12 IMO